src/coreclr/vm/fcall.h

```diff
@@ -362,11 +362,9 @@ class CompletedFCallTransitionState
 
 //==============================================================================================
 // This is where FCThrow ultimately ends up. Never call this directly.
-// Use the FCThrow() macros. __FCThrowArgument is the helper to throw ArgumentExceptions
-// with a resource taken from the managed resource manager.
+// Use the FCThrow() macro.
 //==============================================================================================
 LPVOID __FCThrow(LPVOID me, enum RuntimeExceptionKind reKind, UINT resID, LPCWSTR arg1, LPCWSTR arg2, LPCWSTR arg3);
-LPVOID __FCThrowArgument(LPVOID me, enum RuntimeExceptionKind reKind, LPCWSTR argumentName, LPCWSTR resourceName);
```

Comment:
We can remove the mention of FCThrow from these two comments:

```
src/coreclr/debug/ee/controller.cpp:                // Exceptions raised inside of the EE by COMPlusThrow, FCThrow, etc will not
src/coreclr/vm/threadsuspend.cpp:    // @TODO: we should consider treating this function as an FCALL or HCALL and use FCThrow instead of COMPlusThrow
```

Reply:
I'm leaving mentions of `FCThrow` until it is removed. There are still instances and members of the team that have to work in the debugger should have that knowledge. All of it will be removed, eventually :)